src/main/java/com/uipath/uipathpackage/Utility.java

```diff
@@ -119,37 +50,7 @@ private String getCommand(String... commands) {
         return commandChainBuilder.toString();
     }
 
-    private void copyPluginFiles(@Nonnull TaskListener listener, File tempDir, String pluginJarPath) throws IOException, URISyntaxException {
-        File jar = new File(pluginJarPath);
-        if (!jar.exists()) {
-            // For snapshot plugin dependencies, an IDE may have replaced ~/.m2/repository/…/${artifactId}.hpi with …/${artifactId}-plugin/target/classes/
-            // which unfortunately lacks META-INF/MANIFEST.MF so try to find index.jelly (which every plugin should include) and thus the ${artifactId}.hpi:
-            Enumeration<URL> jellies = getClass().getClassLoader().getResources("index.jelly");
-            while (jellies.hasMoreElements()) {
-                URL jellyU = jellies.nextElement();
-                if (jellyU.getProtocol().equals("file")) {
-                    File jellyF;
-                    try {
-                        jellyF = new File(jellyU.toURI());
-                    } catch (URISyntaxException e) {
-                        e.printStackTrace(listener.getLogger());
-                        throw e;
-                    }
-                    File classes = jellyF.getParentFile();
-                    if (classes.getName().equals("classes")) {
-                        execute(listener, "Copy-Item -Path " + escapePowerShellString(classes.getAbsolutePath() + "\\*") + " -Destination " + escapePowerShellString(tempDir.getAbsolutePath()) + " -Recurse -force");
-                        listener.getLogger().println("Files copied to temp");
-                    }
-                }
-            }
-        } else {
-            listener.getLogger().println("extracting powershell modules to temp folder");
-            extractResourcesToTempFolder(tempDir, jar, listener);
-            listener.getLogger().println("extracted powershell modules to temp folder");
-        }
-    }
-
-    private File getTempDir() throws IOException {
+    private static File getTempDir() throws IOException {
```

Comment:
Why is this method made static? It is only being called from a non static method `extractResourcesToTempFolder`.

Reply:
I have tried to keep all methods which are not being directly invoked. Those methods which needed to mocked by mockito have to be non static as mockito doesn't supports static method mocking